src/Geometry/CTransform.cs

```diff
@@ -76,17 +76,52 @@ private static double[][] GetUniformMatrix(double value)
 			return matrix;
 		}
 
+		/// <summary>Returns a Identity Matrix</summary>
+		private static double[,] GetIdentityMatrix()
```

Comment:
Could you explain a bit more about what one of these is? I assume its a transform/matrix that does nothing?

Reply:
An identity matrix has a 1 along the diagonal. It basically means there is no transformation but its a valid matrix. A zero matrix would not be valid because any transformation added to it will result to nothing. I explained it briefly above